src/js/contexts/MessageContext/MessageContext.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import defaultMessages from '../../languages/default.json';
+
+// opts:
+//   id: message id
+//   messages: (optional) an object of message overrides
+//   values: (optional) currently unused but in the future
+//     will be an object with substitution values for 
+//     positional variables in the message text.
+//   defaultMessage: (optional) default message to use if
+//     the message isn't found elsewhere.
+export const format = (opts, messages) => {
```

Comment:
Can we rename `opts` to `options` to be more explicit? Or am I missing another hidden meaning?

Reply:
Can if you prefer. I tend to use opts kinda like how we tend to abbreviate properties as props. I think I adopted this one from intl.formatMessage() but full spelling is certainly fine.